docs/openapi/api-external-b2b-pa-bundle.yaml

```diff
@@ -1930,34 +2006,220 @@ components:
+        region:
+          $ref: '#/components/schemas/RegionSection'
+        localTax:
+          $ref: '#/components/schemas/LocalTaxSection'
+        excise:
+          $ref: '#/components/schemas/ExciseSection'
+        iban:
+          type: string
+      description: F24 Excise (Accise) object
+    TaxPayerElide:
+      type: object
+      properties:
+        taxCode:
+          pattern: ^([A-Z]{6}[0-9LMNPQRSTUV]{2}[ABCDEHLMPRST]{1}[0-9LMNPQRSTUV]{2}[A-Z]{1}[0-9LMNPQRSTUV]{3}[A-Z]{1})|([0-9]{11})$
+          type: string
+          description: Tax Payer Tax Code
+        person:
+          $ref: '#/components/schemas/PersonData'
+        company:
+          $ref: '#/components/schemas/CompanyData'
+        relativePersonTaxCode:
+          pattern: ^([A-Z]{6}[0-9LMNPQRSTUV]{2}[ABCDEHLMPRST]{1}[0-9LMNPQRSTUV]{2}[A-Z]{1}[0-9LMNPQRSTUV]{3}[A-Z]{1})|([0-9]{11})$
+          type: string
+          description: It is the tax code of a relative of the main tax payer
+        id:
+          pattern: ^[A-Z0-9]{2}$
+          type: string
+          description: identification code
+    TreasuryRecord:
+      allOf:
+        - $ref: '#/components/schemas/IncludeNotificationCost'
+      type: object
+      properties:
+        type:
+          pattern: ^[A-Z]$
+          type: string
+          description: type of treasury
+        id:
+          pattern: ^[A-Z0-9]{17}$
+          type: string
+          description: identification code of the element
+        taxType:
+          pattern: ^[A-Z0-9]{4}$
+          type: string
+          description: identification code of the type of tax
+        year:
+          pattern: ^[1-2][0-9]{3}$
+          type: string
+          description: referance year
+        debit:
+          pattern: ^[0-9]{3,15}$
+          type: string
+          description: debit amount of the record
+      description: Treasury Record object
+    TreasuryAndOtherSection:
+      type: object
+      properties:
+        office:
+          pattern: ^[A-Z0-9]{3}$
+          type: string
+          description: identification code of the office
+        document:
+          pattern: ^\d{11}$
+          type: string
+          description: identification code of the document
+        records:
+          type: array
+          items:
+            $ref: '#/components/schemas/TreasuryRecord'
+          description: Treasury Records
+      description: Treasury ans Other Section (Sezione Erario e Altro) object
+    F24Elid:
+      type: object
+      properties:
+        taxPayer:
+          $ref: '#/components/schemas/TaxPayerElide'
+        treasury:
+          $ref: '#/components/schemas/TreasuryAndOtherSection'
+        bankAccountNumber:
+          type: string
+          description: IBAN bank account number
+        abi:
+          type: string
+          description: abi identification code
+        bankId:
+          type: string
+          description: identification code of the bank
```

Comment:
Queste sono da eliminare perché saranno valorizzate in fase di pagamento in banca